webapp/src/main/java/ar/edu/itba/paw/webapp/controllers/old/OldStudentController.java

```diff
@@ -0,0 +1,783 @@
+//			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("addGrade_success",
+//					new Object[] {},
+//					Locale.getDefault()));
+//		}
+//
+//		return new ModelAndView("redirect:/courses/" + c + "/students");
+//	}
+
+//	@RequestMapping(value = "/students/{docket}/delete", method = RequestMethod.POST)
+//	public ModelAndView removeStudent(@PathVariable final Integer docket, RedirectAttributes redirectAttributes,
+//									  @ModelAttribute("user") UserSessionDetails loggedUser) {
+//
+//		if (!loggedUser.hasAuthority("DELETE_STUDENT")
+//				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
+//			LOGGER.warn("User {} tried to delete a student and doesn't have DELETE_STUDENT authority [POST]", loggedUser.getDni());
+//			return new ModelAndView(UNAUTHORIZED);
+//		}
+//		studentService.deleteStudent(docket);
+//		redirectAttributes.addFlashAttribute("alert", "success");
+//		redirectAttributes.addFlashAttribute("message", messageSource.getMessage("deleteStudent_success",
+//				null,
+//				Locale.getDefault()));
+//
+//		return new ModelAndView("redirect:/students");
+//	}
+
+//	@RequestMapping("/students/{docket}/edit")
+//	public ModelAndView editStudent(
+//			@PathVariable final Integer docket,
+//			@ModelAttribute("studentForm") final StudentForm studentForm,
+//			final RedirectAttributes redirectAttributes,
+//			@ModelAttribute("user") UserSessionDetails loggedUser) {
+//
+//		if (!loggedUser.hasAuthority("EDIT_STUDENT")
+//				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
+//			LOGGER.warn("User {} tried to edit a student and doesn't have EDIT_STUDENT authority [GET]", loggedUser.getDni());
+//			return new ModelAndView(UNAUTHORIZED);
+//		}
+//
+//		final ModelAndView mav = new ModelAndView("addUser");
+//
+//		HTTPErrorsController.setAlertMessages(mav, redirectAttributes);
+//		Student student = studentService.getByDocket(docket);
+//
+//		if (student == null){
+//			redirectAttributes.addFlashAttribute("alert", "danger");
+//			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("editStudent_fail",
+//					null,
+//					Locale.getDefault()));
+//			return new ModelAndView("redirect:/students");
+//		}
+//
+//		if (!redirectAttributes.getFlashAttributes().containsKey("justCalled")) {
+//			studentForm.loadFromStudent(student);
+//		}
+//
+//		mav.addObject("docket", docket);
+//		mav.addObject("student", student);
+//		mav.addObject("section2", "edit");
+//		return mav;
+//	}
+
+//	@RequestMapping(value = "/students/{docket}/edit", method = RequestMethod.POST)
+//	public ModelAndView editStudent(@PathVariable final Integer docket,
+//									@Valid @ModelAttribute("studentForm") StudentForm studentForm,
+//									final BindingResult errors,
+//									RedirectAttributes redirectAttributes,
+//									@ModelAttribute("user") UserSessionDetails loggedUser) {
+//
+//		if (!loggedUser.hasAuthority("EDIT_STUDENT")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)